package/src/Slider.tsx

```diff
@@ -240,12 +240,12 @@ const SliderComponent = (
   const lowerLimit =
     !!localProps.lowerLimit || localProps.lowerLimit === 0
       ? localProps.lowerLimit
-      : LIMIT_MIN_VALUE;
+      : Platform.select({ web: localProps.minimumValue, default: LIMIT_MIN_VALUE });
```

Comment:
Currently, for now this is alright, however, we should aim for unified approach for all platforms.
So the options (for future) are:
1. Remove `LIMIT_MIN_VALUE` approach in favor of the minimum value
2. Use the number range threshold for every.

This is just a thought for future, doesn't concern this PR.